charts/guac/values.yaml

```diff
@@ -17,6 +17,7 @@ imagePullSecrets:
 ## @param guac.guacImage.workingDir Working Directory for GUAC
 ## @param guac.common.env [string] common environment variables apply to all guac services
 ## @param guac.common.tolerations [string] common tolerations apply to all guac services
+## @param guac.configMap.enabled Whether to create the guac-cm configMap
```

Comment:
It's not clear what the guac-cm config map is here.

Reply:
The configmap contains the guac cli flag values.

The variable here allow one to not create the configmap and pass the flags individually into respective guac component or create their own config map.